### PR TITLE
Add support for Dart Sass

### PR DESCRIPTION
**Pull request by duggiemitchell**
## Cute animal pic
418352e6ff99a49092c35c63d3c97674--ps

Fixes: #342 
Fixes: #341 

## Description:

- [x] Set node-sass as default compiler
- [x] Add support for Dart Sass
- [x] Update Docs
- [x] Update Changelog

### CODE REVIEW

**Comment by duggiemitchell** [comment]
[Update Herman to work with Dart-Sass.](https://trello.com/c/USlvmZry/11-update-herman-to-work-with-dart-sass)

**Comment by mirisuzanne** [comment]
What's the best way to test this? Can we switch the internal herman documentation to use dart-sass, or should I point another project at my local repo on this branch? 

The main potential issue I'm seeing is how we handle `sass.includes` moving forward. `@import` still exists in Dart Sass, but much of it's (Sass) functionality is being deprecated (it will still support plain-CSS imports). I think, in order to support the transition (and the bonus use-case of plain-css imports), we'll need a new `sass.use` array – allowing authors to make the switch explicitly.

File: docs/config-scale.html [hunk 1]
```diff
@@ -988,12 +988,12 @@ <h3 class="item-subtitle">
     </td>
     <td class="size-value">
       <code>
-        0.9rem
+        .9rem
```
Comment (jgerigmeyer) [inline docs/config-scale.html:991]: @mirisuzanne I'm not 100% sure what's causing this change in leading zeros... I assume an upgrade in node-sass version?

File: scss/initial/_root.scss [hunk 2]
```diff
@@ -50,7 +50,7 @@ mark {
   #{$link} {
     color: color('action');
     text-decoration-color: color('underline');
-    text-decoration-skip: ink;
+    text-decoration-skip: auto;
```
Comment (jgerigmeyer) [inline scss/initial/_root.scss:53]: @mirisuzanne Note this change -- there was a warning that the spec had changed.

File: docs/config-scale.html (hunk 1 above)
Comment (mirisuzanne) [reply docs/config-scale.html:991]: yeah, not sure exactly - but likely just a (non-breaking) change between Sass implementations.

File: docs/style-banner.html [hunk 3]
```diff
@@ -583,15 +583,14 @@ <h3 class="item-subtitle">
 
       
       
-      <pre class="hljs-pre"><code class="lang-css">[data-region=&#39;banner&#39;] {
+      <pre class="hljs-pre"><code class="lang-css">[data-region=banner] {
```
Comment (jgerigmeyer) [inline docs/style-banner.html:586]: @mirisuzanne Whoa -- these changes are weird.

Screenshot 2020-04-24 18 30 14

Comment (mirisuzanne) [reply docs/style-banner.html:586]: Yeah, that's valid - and has the same meaning in CSS. Dart Sass is more aggressive about optimizing output like that.

Comment (jgerigmeyer) [reply docs/style-banner.html:586]: 🤯

**Pull request merged**